apps/openmw/mwworld/scene.cpp

```diff
@@ -43,22 +43,34 @@ namespace
     }
 
     void updateObjectLocalRotation (const MWWorld::Ptr& ptr, MWPhysics::PhysicsSystem& physics,
-                                    MWRender::RenderingManager& rendering)
+                                    MWRender::RenderingManager& rendering,
+				    int prio = 0)
```

Comment:
it's int because I wanted to leave room for an eventual 3rd order even if
it's extremely unlikely.
We can have "rotation_order" instead if you prefer, prio was for axis
priority in the rotations of course.

2015-09-15 21:20 GMT+02:00 Mingun <notifications@github.com>:

> In apps/openmw/mwworld/scene.cpp
> <https://github.com/OpenMW/openmw/pull/760#discussion_r39553098>:
>
> > @@ -43,22 +43,34 @@ namespace
> >      }
> >
> >      void updateObjectLocalRotation (const MWWorld::Ptr& ptr, MWPhysics::PhysicsSystem& physics,
> > -                                    MWRender::RenderingManager& rendering)
> > +                                    MWRender::RenderingManager& rendering,
> > +				    int prio = 0)
>
> prio has bool semantic, so, I think, it must be of type bool. Also, name
> prio is cryptic. I think its name must reflect multiplication order.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenMW/openmw/pull/760/files#r39553098>.
>